source/reference/minio-mc/mc-tag-list.rst

```diff
@@ -72,6 +72,13 @@ Parameters
 
       mc tag list myminio/mybucket/object.txt
 
+.. mc-cmd:: --recursive, r
+   :optional:
+
+   .. versionadded:: RELEASE.2023-05-04T18-10-16Z
+
+   Recursively lists the tags for all objects at the specified :mc:`ALIAS <mc tag list ALIAS>`.
```

Comment:
For an entire deployment? 
Wouldn't you want to do this at a bucket or prefix instead of a whole deployment?

Reply:
Erg. Since we specify `ALIAS` as the parameter, but expect the ALIAS to be the full path. Probably for commands like this we should either split it to ALIAS/PATH or call it TARGET. But then we end up with all kinds of divergences. Blah.